docs/watch_tutorial/index.md

```diff
@@ -0,0 +1,362 @@
+
+![Casio](media/casio.jpg)
+
+Let's add this make this the gadget preview model while we're at it! in `manifest.webmanifest`, under `icons`, change to:
+```webmanifest
+	"icons": [
+		{
+			"src": "models/watch.glb",
+			"type": "model/gltf-binary"
+		}
+	],
+```
+**important** - remember to add the watch (and all gltf models) to your **`dist/models`** folder, not `src/models`! gltf models do not get copied over as part of the compilation flow - the path is relative from `dist/manifest.webmanifest`.
+
+Now that our gadget preview is all set up, let's add the watch to the gadget itself:
+```tsx
+<div>
+	<AvGrabbable dropOnHooks={true}>
+		<AvSphereHandle radius={0.1} />
+		<AvTransform uniformScale={0.2}>
+			<AvPanel interactive={true} >
+				<AvTransform translateX={.063} translateY={-.28} translateZ={-.023}>
+					<AvModel uri="./models/watch.glb" scaleToFit={{ x: 0.35, y: 0.35, z: 0.35 }} />
+				</AvTransform>
+				<div className="Watch">
+				Hello Aardvark!
+				</div>
+			</AvPanel>
+		</AvTransform>
+	</AvGrabbable>
+</div>
+```
+
+Not *too* much happening here. Inside AvPanel, we've added a `<AvTransform>` component and inside *it*, an `<AvModel>` component. We're using the `<AvTransform>` component to place the model where we want it in the gadget and `<AvModel>`'s `scaleToFit` property to lock the model's scale into an (imaginary) box of a certain size. The values are a result of trial and error - our main concern is having the watch in a reasonable place and setting the digits' size and location with CSS.
+
+# Step 4: _Time for Action_ - Making it interactive
+
+Alright, so we've got a watch; time (heh) to make it tick! 
+In the spirit of old school digital watches, let's have a `time` display and - when the screen is "pressed" - a `date` display as well. We'll start by adding a "watchDisplay" string and an instance of a "watchDisplayType" enum to our state interface, to manage the display state:
+
+```tsx
+enum watchDisplayType {
+	Time,
+	Date
+}
+
+interface WatchState {
+	watchDisplayTypeInstance: watchDisplayType;
+	watchDisplay: string;
+}
+```
+
+In the `WatchGadget` class, we'll: 
+* initialize this state in the constructor 
+* add class methods to keep time & update the display state
+
+```tsx
+
+class WatchGadget extends React.Component<{}, WatchState> {
+
+    constructor(props: any) {
+			super(props);
+			this.state =
+			{
+				watchDisplayTypeInstance: watchDisplayType.Time,
+				watchDisplay: ""
+			};
+    }
+		
+    componentDidMount() {
+			setInterval(this.getTime, 1000);
+    }
+
+    @bind public getTime() {
```

Comment:
This can probably be private.

I've taken to putting the decorators (@bind) on the preceding line so public/private it leftmost on the line that declares the function.